documentation/design-docs/dotnet-tools.md

```diff
@@ -1,124 +1,801 @@
-#Dotnet Diagnostic Tools CLI Design
+# Dotnet Diagnostic Tools CLI Design
 
-Capturing some comments from https://github.com/dotnet/diagnostics/issues/85 as a starting point
-we can edit from. This needs cleanup.
+# User workflows
 
-@davidfowl wrote:
+These are some quick examples of the work we'd expect a .Net developer to want to get a feel for the design. A more complete reference follows.
 
-@shirhatti We're thinking it should be part of dotnet-collect since all of the flags and infrastructure would likely be the same and it should have a "top" like interface. 
+### Ad-hoc Health Monitoring / 1st Level Performance Triage
 
-![image](https://user-images.githubusercontent.com/95136/49297973-09575400-f471-11e8-99ec-823e616eafa2.png)
 
-We'll need to decide what things we show (aggregations and counters)
+**Seeing performance counter values that refresh periodically in-place**
 
+    > dotnet tool install -g dotnet-counters
+    You can invoke the tool using the following command: dotnet-counters
+    Tool 'dotnet-counters' (version '1.0.0') was successfully installed.
+    > dotnet counters monitor --process-id 1902 Microsoft-Windows-DotNETRuntime Microsoft-AspNet
+    
+    Microsoft-Windows-DotNETRuntime: 
```

Comment:
Quick question on this -- Are we planning on naming this ```Microsoft-Windows-DotNETRuntime``` instead of ```System.Runtime```? In my PR I just named it ```System.Runtime``` per @vancem's suggestion, and I realize that it's easy to change it before we officially ship, but I was just wondering if we're settled on this name.

Reply:
I'll change the name in the spec to match whatever you and Vance agreed upon. I used Microsoft-Windows-DotNETRuntime because that is what it was named as of a few weeks ago, but I didn't particularly like it.